docs/instructions.md

```diff
@@ -0,0 +1,65 @@
+- [NHS - Posture Tips for Laptop Users](https://www.nhs.uk/live-well/healthy-body/posture-tips-for-laptop-users/)
+
+## "Unknown Posture" detected?
+
+If pose estimation fails to detect your whole body, we bring this to your attention in the form of <span style="color:grey">Grey</span> lines - if you see this, please adjust your camera or your own positioning until you are clearly detectable in the video feed. In the Settings we have a slider for "Confidence Threshold" which you can tune to modify this - if you feel like you're being presented with an <span style="color:grey">"Unknown Posture"</span> even when you are in an optimal position, decrease this value a bit and that should improve your experience.
+
+<div align="center">
+    <img src="images/instructions/unknown_posture.png" width="500"/>
+</div>
+
+## Achieving PosturePerfection!
+
+When you are happy with a posture that you have adopted and have clicked "Set Ideal Posture", your detected posture will be highlighted in <span style="color:green">Green</span> as you are then in a position where you have achieved PosturePerfection! Carry on with your work, and the system will let you know when we detect you have strayed from this ideal posture. See below for more details to ensure you maximise your time in PosturePerfection!
+
+<div align="center">
+    <img src="images/instructions/good_posture.png" width="500"/>
+</div>
+
+## Realtime Guidance to PosturePerfection!
+
+When we determine that your posture is no longer close to your ideal posture, arrows are drawn to guide you back to a better posture. Follow these arrows back to your perfect posture! We provide a configurable "Slouch Sensitivity" in the Settings. We have chosen a default value which worked well during development, but we are aware you might want to change this. Increase this if you would like the system to be more strict with you, such that minor movements from your ideal posture will be detected. However, if you would like the system to be a bit more lenient, feel free to decrease this value.
+
+<div align="center">
+    <img src="images/instructions/bad_posture.png" width="500"/>
+</div>
+
+## Setting up Notifications
+
+We are aware that some of you may choose to setup PosturePerfection on a separate device from your work device. Therefore, we have incorporated the sending of notifications from PosturePerfection to another Linux device on the same network when the system detects you have started slouching! Note that this isn't an exclusive feature - you can even receive these notifications on the same device on which you are running PosturePerfection!
+
+To do this, ensure you have `simple-remote-notify-send` installed. Find the latest release and more information [here](https://github.com/ESE-Peasy/simple-remote-notify-send/releases/tag/v1.0.0). Run the `PosturePerfectionReceiver` on the device where you want to receive notifications, and then start `PosturePerfection`.
```

Comment:
Could we make it completely clear that this is part of ***our*** software? It think that might make users more comfortable in using it and of course clarifies that for getting marks. (Possibly even add install instructions here, what do you think?)

Reply:
Yeah good idea, I'll add some more info specifying that we have made `simple-remote-notify-send` ourselves. As for install instructions, I'm not so sure - there's nothing to add that isn't covered in the release installation instructions right?